### ISSUE TITLE
Notice component fails 2.5.8 Target Size (Minimum)

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk_publishing_components#3439]
The [Notice component](https://components.publishing.service.gov.uk/component-guide/notice) fails [2.5.8 Target Size (Minimum)](https://w3c.github.io/wcag/understanding/target-size-minimum.html).

Note: 2.5.8 is part of the new draft WCAG 2.2 criteria. The latest standard is WCAG 2.1. WCAG 2.2 isn't published yet, so this is potentially subject to change.

The Notice component fails when links are on separate lines (like in the ‘Without title’ example). That's because those links are only 23.5px high and have no further spacing between them. They need to be at least 24px high or have additional spacing that adds up to 24px in order to pass.

Screenshot showing the dimensions of the link:

<img width="421" alt="Screenshot of notice component without title" src="https://github.com/alphagov/govuk_publishing_components/assets/108893/29ce601d-9728-4e79-bd71-dd25e3a2ab15">

**Comment by jon-kirwan** [comment]
@selfthinker I wonder if we need to make any changes here.

The Notice component example includes the following markup (`<br>` elements are included to produce a line breaks).
```
<p>
Scheduled to publish at 8am on 25 April 2019<br />
<a href="change-date">Change date</a><br />
<a href="stop-scheduled-publishing">Stop scheduled publishing</a>
</p>
```

However, from several instances I've looked at, publishers often seem to either add links in inline text or format them with a list e.g. -

- [Code of practice for the exercise of stop and search powers ](https://www.gov.uk/government/publications/code-of-practice-for-the-exercise-of-stop-and-search-powers)
- [About the Document Checking Service pilot scheme](https://www.gov.uk/guidance/apply-for-the-document-checking-service-pilot-scheme)
- [VAT Mini One Stop Shop: Non-Union Return](https://www.gov.uk/government/publications/vat-mini-one-stop-shop-non-union-return)
- [NHS COVID-19 app](https://www.gov.uk/government/collections/nhs-covid-19-app)
- [How to classify trade marks](https://www.gov.uk/government/publications/how-to-classify-trade-marks)
- [Withdrawn: School inspection handbook](https://www.gov.uk/government/publications/school-inspection-handbook-from-september-2015)
- [Legal definition of waste guidance](https://www.gov.uk/government/publications/legal-definition-of-waste-guidance)
- [The Grand Challenges](https://www.gov.uk/government/publications/industrial-strategy-the-grand-challenges/industrial-strategy-the-grand-challenges).

Maybe the example should be updated to format the two links in a bulleted list (which, I think, does then satisfy two exceptions to the success criterion):

> Spacing: Undersized targets (those less than 24 by 24 CSS pixels) are positioned so that if a 24 CSS pixel diameter circle is centered on the [bounding box](https://w3c.github.io/wcag/understanding/target-size-minimum.html#) of each, the circles do not intersect another target or the circle for another undersized target;
...
Inline: The target is in a sentence or its size is otherwise constrained by the line-height of non-target text;

**Comment by selfthinker** [comment]
@jon-kirwan, yes, having a list of links in this example will fix this. It would also be generally better as a proper list is more semantic than the sort of 'fake list' we have there now.
